tests/pr_test.go

```diff
@@ -44,3 +45,33 @@ func TestRunUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestPrivateInSchematics(t *testing.T) {
+	t.Parallel()
+
+	const region = "us-south"
```

Comment:
Why hard code to us-south ? The extended quotas we have are per region, so we need to spread our tests around to all the regions in https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/common-go-assets/cloudinfo-region-secmgr-prefs.yaml